packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -202,7 +202,7 @@ export interface ICreateFileResponse {
 export interface IVersionedValueWithEpoch {
     value: any;
     fluidEpoch: string,
-    version: 2,
+    version: 3,
```

Comment:
I assume there is no way to use persistedCacheValueVersion here. Maybe add comment that it's same as persistedCacheValueVersion?